engine/src/cmd/unit_util_generic.cpp

```diff
@@ -78,8 +78,8 @@ bool hasDockingUnits( const Unit *my_unit )
 
 int getPhysicsPriority( Unit *un )
 {
-    static const bool FORCE_TOP_PRIORITY = XMLSupport::parse_bool(
-        vs_config->getVariable( "physics", "priorities", "force_top_priority", "false" ) );
+    static const bool FORCE_TOP_PRIORITY = true; //XMLSupport::parse_bool(
+        //vs_config->getVariable( "physics", "priorities", "force_top_priority", "false" ) );
```

Comment:
This could probably just `bool FORCE_TOP_PRIORITY` or `const bool FORCE_TOP_PRIORITY` for its type; it's only used in this function. `static` is unnecessary.

Not a blocker, just noting it.

Reply:
If it was just `bool FORCE_TOP_PRIORITY`, it would probably read the setting correctly, after vs_config is initialized properly with the values from the file. Maybe I should just do that.

Reply:
Took out the `const` clause. Now it reads its value correctly from the config file.